diff --git a/node_modules/@earendil-works/pi-coding-agent/dist/config.js b/node_modules/@earendil-works/pi-coding-agent/dist/config.js
index 4600b23..95262e9 100644
--- a/node_modules/@earendil-works/pi-coding-agent/dist/config.js
+++ b/node_modules/@earendil-works/pi-coding-agent/dist/config.js
@@ -391,7 +391,7 @@ const piConfigName = pkg.piConfig?.name;
 export const PACKAGE_NAME = pkg.name || "@earendil-works/pi-coding-agent";
 export const APP_NAME = piConfigName || "pi";
 export const APP_TITLE = piConfigName ? APP_NAME : "π";
-export const CONFIG_DIR_NAME = pkg.piConfig?.configDir || ".pi";
+export const CONFIG_DIR_NAME = ".pi-studio";
 export const VERSION = pkg.version || "0.0.0";
 // e.g., PI_CODING_AGENT_DIR or TAU_CODING_AGENT_DIR
 export const ENV_AGENT_DIR = `${APP_NAME.toUpperCase()}_CODING_AGENT_DIR`;
